src/sentry/utils/committers.py

```diff
@@ -191,6 +191,17 @@ def get_event_file_committers(project, event, frame_limit=25):
     if not app_frames:
         app_frames = [frame for frame in frames][-frame_limit:]
 
+    # Java stackframes don't have an absolute path in the filename key.
+    # That property is usually just the basename of the file. In the future
+    # the Java SDK might generate better file paths, but for now we use the module
+    # path to approximate the file path so that we can intersect it with commit
+    # file paths.
+    sdk = event.interfaces.get('sdk')
+    if sdk and sdk.name in ('sentry-java', 'sentry.java'):
```

Comment:
@bretthoerner Does the java-sdk always report as `sentry-java` or does it sometimes use a `.`?

Reply:
It's always `sentry-java`, but there is also `sentry-android` and many deprecated SDKs that are still in large use (going by Snuba data) like `Raven-Java`, `raven-java` and more. Is there a reason just using `platform = java` would be bad?

Reply:
That said using those 4 will cover like 95% probably.

Reply:
I'm happy with platform = java :+1: